docs/tutorials/getting-started/intro.md

```diff
@@ -7,7 +7,7 @@ Welcome to Great Expectations! This tutorial will help you set up your first loc
 :::warning
 The steps described in this tutorial assume you are installing Great Expectations version 0.13.8 or above.
 
-For a tutorial for older versions of Great Expectations, please see older versions of this documentation.
+For a tutorial for older versions of Great Expectations, please see older versions of this documentation, which can be found at [https://docs.greatexpectations.io/](https://docs.greatexpectations.io/en/latest/).
```

Comment:
I would either update the link to lead directly to the tutorial on the old docs site, or update the phrasing of the line (Like "For documentation on older versions of GE...")

Reply:
oh man, thank you for catching this too. Since we are eventually going to be migrating this page to the `https://docs.greatexpectations.io/` site, I'll change the wording to "here" as well